prime-dotnet-webapi/Controllers/UtilController.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+
+using Prime.Auth;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UtilController : PrimeControllerBase
+    {
+        [HttpGet("gccollect", Name = nameof(GCCollect))]
+        [Authorize(Roles = Roles.ManageEnrollee)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
```

Comment:
Is this the correct Role to require?

Reply:
I asked for manager admin.

Reply:
Since we don't have a way to access this endpoint through the app is there a point in adding perm? Like i guess if you export your token and use it through postman you can get this to work?